### PR TITLE
use official online container

### PR DESCRIPTION
**Pull request by jeffnvidia**
## Summary
Based on PR 31
This MR is for using the official latest NeMo version

## Test Plan
Test by @amaslenn
CI
Test by @jeffnvidia
2.1 Slurm command generation
$ cloudaix --mode run --system_config_path conf/v0.6/general/system/israel_1.toml --test_scenario_path conf/v0.6/general/test_scenario/llama/llama.toml

### CODE REVIEW

**Comment by srinivas212** [comment]
Needs PR template populated.

**Comment by srinivas212** [comment]
Pl rebase directly on main.

**Comment by jeffnvidia** [comment]
> Pl rebase directly on main.

It was based on another PR of mine so that everything can be merged together

File: tests/test_slurm_system.py
Comment (amaslenn) [inline tests/test_slurm_system.py:1]: I don't think these changes belong to this PR.

**Review by amaslenn — CHANGES_REQUESTED** [review]

File: tests/test_slurm_system.py
Comment (jeffnvidia) [reply tests/test_slurm_system.py:1]: is this still relevant ?

Comment (amaslenn) [reply tests/test_slurm_system.py:1]: No

**Review by amaslenn — APPROVED** [review]

**Comment by lappazos** [comment]
@srinivas212 @TaekyungHeo please merge? was tested completlely

**Pull request merged**